sparkle/src/components/DataTable.tsx

```diff
@@ -185,7 +206,7 @@ export function DataTable<TData extends TBaseData>({
                   return null;
                 }
                 return (
-                  <DataTable.Cell key={cell.id}>
+                  <DataTable.Cell column={cell.column} key={cell.id}>
```

Comment:
OOC: what's this used for?

Reply:
it's used to get the size (meta) properties from columnDef